sentry/sentry.conf.example.py

```diff
@@ -209,13 +209,24 @@
     }
 )
 
+# Hacky stuff
+class SentryOptionsMapper(object):
+    def __init__(self, option_name):
+        self._option_name = option_name
+
+    def __get__(self, instance, owner):
+        if not hasattr(self, '_option_val'):
+            from django.conf import settings
+            self._option_val = settings.SENTRY_OPTIONS.get(self._option_name)
+        return self._option_val
```

Comment:
@BYK Wondering why we need caching here. If something in `settings.SENTRY_OPTIONS` changes for any reason, we won't see that change.

Reply:
It's immutable after boot. And this import isn't free. In theory, this is just memoization.

Reply:
@mattrobenolt good point about the import, I agree this way makes sense